BlueprintToCpp/Program.cs

```diff
@@ -87,16 +87,28 @@ public static async Task Main(string[] args)
                 return;
             }
 
+            string zlibPath = config.ZlibPath;
+            if (string.IsNullOrEmpty(zlibPath) || zlibPath.Length < 1)
+            {
+                Console.WriteLine("Missing \"ZlibPath\" in the config.json file.");
+#if FALSE
+                return;
+#endif
```

Comment:
```suggestion
```
Remove this since we don't want it to stop the program if zlib is missing - zlib is optional